### ISSUE TITLE
Clean up of storyscript repositories

### ISSUE THREAD

**Issue by williammartin** [storyscript/community#13]
There are a bunch of repositories that seem suspicious in their use:

https://github.com/storyscript/welcome
https://github.com/storyscript/stack-compose
https://github.com/storyscript/syntax
https://github.com/storyscript/components-v2
https://github.com/storyscript/bootstrap
https://github.com/storyscript/http-old
https://github.com/storyscript/ui
https://github.com/storyscript/websockets
https://github.com/storyscript/platform-rest
https://github.com/storyscript/homebrew-core
https://github.com/storyscript/platform-rabbitmq
https://github.com/storyscript/hub-sync-temp
https://github.com/storyscript/test-jira-sync
https://github.com/storyscript/image-builder (super core to the platform clearly)

How many of these can we delete?

**Comment by wilzbach** [comment]
Please don't delete https://github.com/storyscript/syntax. It's actively used for e.g. the vscode and atom plugin.

**Comment by stevepeak** [comment]
@williammartin please proceed at deleting repositories that are no longer in use. Let's back them up just to make sure

**Comment by williammartin** [comment]
I actually don't have the privileges to delete these.

**Comment by wilzbach** [comment]
Okay I went through all repositories and I removed the empty ones. However, I did _not_ remove archived ones. Imho there's no cost in keeping them around for a little longer.
In this spirit I also archived more when there was actual code:

- https://github.com/storyscript/platform-rabbitmq -> archived
- https://github.com/storyscript/hub-sync-temp -> archived

Furthermore:

- homebrew-core is actively used: https://github.com/storyscript/cli/blob/master/scripts/update_brew.sh

If we decide that we want to get rid of the archived repositories, I propose that we simply move them into a special org (e.g. `storyscript-archive`) as this still allows them to be found easily by everyone if it's ever desired.
